### DISCUSSION TITLE
Unnecessary components and pages being requested.

### DISCUSSION THREAD

**Discussion by praveshishere** [nuxt/nuxt#16979] [Questions] [answered]
When I load my homepage, the network tab shows that all other components and pages of my site are also being requested, the response to the requests is blank. still, the size of the requests goes around 1 to 10 KBs.
The size is small but the number of files makes it huge and Increases the blocking time. 

The same also happens with other pages of the site

Does anyone have any idea why is this happening or how to resolve this?

**Comment by danielroe** [comment, marked as answer]
This is already resolved on the edge channel: https://v3.nuxtjs.org/guide/going-further/edge-channel/

**Reply by praveshishere** [reply]
I tried installing the edge version.

I am getting an error `nuxi not found`.
Installing nuxi manually gives the error `Cannot start nuxt: Cannot find any nuxt version from ...`

**Reply by praveshishere** [reply]
however, it works with yarn, no idea why it didn't work with npm
